### PR TITLE
Added `SHARELATEX_SAML_DECRYPTION_CERT` environment variable

### PR DESCRIPTION
**Pull request by mserranom**
## Description

Public counterpart of `decryptionPvk`, required to fetch SAML metadata.

## Contributor Agreement

- [x] I confirm I have signed the [Contributor License Agreement](https://github.com/overleaf/overleaf/blob/master/CONTRIBUTING.md#contributor-license-agreement)

### CODE REVIEW

File: settings.coffee [hunk 1]
```diff
@@ -438,6 +438,7 @@ if process.env["SHARELATEX_SAML_ENTRYPOINT"]
 			callbackUrl: process.env["SHARELATEX_SAML_CALLBACK_URL"]
 			issuer: process.env["SHARELATEX_SAML_ISSUER"]
 			decryptionPvk: process.env["SHARELATEX_SAML_DECRYPTION_PVK"]
+      decryptionCert: process.env["SHARELATEX_SAML_DECRYPTION_CERT"]
```
Comment (gary-kim) [inline settings.coffee:441]:
```suggestion
			decryptionCert: process.env["SHARELATEX_SAML_DECRYPTION_CERT"]
```

**Review by ShaneKilkelly — APPROVED** [review]
Looks good, but we need to change the spaces to tabs, as noted in the comment on the PR :+1:

File: settings.coffee (hunk 1 above)
Comment (mserranom) [reply settings.coffee:441]: Thanks! Would you mind signing the [Contributor Agreement](https://docs.google.com/forms/d/19JsUGMlhM5LJ0mA4Vj_guMiSoX1pNg_-KvvZT5-H0LQ) before proceeding to include your suggestion?

https://docs.google.com/forms/d/19JsUGMlhM5LJ0mA4Vj_guMiSoX1pNg_-KvvZT5-H0LQ

Comment (gary-kim) [reply settings.coffee:441]: Sorry, I don't sign CLAs. Didn't realize the review was gonna make a problem for that. If it helps, I'm licensing it under CC0.
If it's that much of a problem, let me know an I'll sign it for this time.

**Pull request merged**